src/porepy/applications/md_grids/fracture_sets.py

```diff
@@ -65,13 +65,12 @@ def orthogonal_fractures_3d(size: pp.number) -> list[pp.PlaneFracture]:
     return [pp.PlaneFracture(pts[i]) for i in range(3)]
 
 
-def benchmark_regular_2d_fractures(
-    size: Optional[pp.number] = 1,
-) -> list[pp.LineFracture]:
-    """Return a list of regular fractures as used in the 2d benchmark study.
+def benchmark_regular_2d_fractures(size: pp.number = 1) -> list[pp.LineFracture]:
```

Comment:
@keileg, should we rename to benchmark_2d_case_3 for consistency with the new function below?

Reply:
Sounds like a good idea.